oic.r.cred.raml

```diff
@@ -95,8 +118,19 @@ traits:
         description: The request is invalid.
 
   post:
+
```

Comment:
Oh - from just the "privatedata" example I presume? Yeah - this passed schema validation since schema can't express contextual dependencies. 

There are other examples for cred (in the "examples" directory). So I'd advocate a separate change/PR to fixup all cred examples.